packages/i/imagemagick/xmake.lua

```diff
@@ -0,0 +1,96 @@
+package("imagemagick")
+    set_homepage("https://imagemagick.org/script/index.php")
+    set_description("ImageMagick is a FOSS software suite for modifying images. This does NOT provide any of the utilities. It installs the C/C++ Libraries.")
+    add_urls("https://download.imagemagick.org/ImageMagick/download/ImageMagick-$(version).tar.gz")
+    add_versions("7.0.11-13", "6c162d7cbd7b80968a6d083d39eb18d9c9bbd49f500f7f49c9a5dcc0fc36a03b")
+    add_deps("pkg-config")
+    add_configs("bzlib", {description = "Enable bzip2 support.", default = false, type = "boolean"})
+    add_configs("exr", {description = "Enable exr support.", default = false, type = "boolean"})
+    add_configs("fftw", {description = "Enable fftw support.", default = false, type = "boolean"})
+    add_configs("fontconfig", {description = "Enable fontconfig support.", default = false, type = "boolean"})
+    add_configs("freetype", {description = "Enable freetype support.", default = false, type = "boolean"})
+    add_configs("jpeg", {description = "Enable jpeg support through libjpeg.", default = true, type = "boolean"})
+    add_configs("lzma", {description = "Enable LZMA support.", default = false, type = "boolean"})
+    add_configs("openjpeg", {description = "Enable jpeg support through openjpeg.", default = false, type = "boolean"})
+    add_configs("png", {description = "Enable png support.", default = true, type = "boolean"})
+    add_configs("raw", {description = "Enable raw image support.", default = false, type = "boolean"})
+    add_configs("tiff", {description = "Enable tiff support.", default = false, type = "boolean"})
+    add_configs("threads", {description = "Enable threading support.", default = false})
+    add_configs("xml", {description = "Enable XML support.", default = false, type = "boolean"})
+    add_configs("webp", {description = "Enable webp support.", default = false, type = "boolean"})
+    add_includedirs("include/ImageMagick-7/")
+    add_links("MagickCore-7.Q16", "MagickWand-7.Q16", "Magick++-7.Q16")
```

Comment:
> MagickWand depends on MagickCore, and Magick++ is separate. 

So it should be:

    add_links("MagickWand-7.Q16", "MagickCore-7.Q16", "Magick++-7.Q16")